plugins/UM3NetworkPrinting/src/Models.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+from collections import namedtuple
+
+ClusterMaterial = namedtuple('ClusterMaterial', [
+    'guid',
+    'material',
+    'brand',
+    'version',
+    'color',
+    'density'
+])
+
+LocalMaterial = namedtuple('LocalMaterial', [
+    'GUID',
```

Comment:
why is GUID capitalised here, but not in ClusterMaterial?

Reply:
Because the keys are different between the networked printer response and the local registry. Don’t ask why...

Reply:
But wh... oh, ok.

Perhaps that should be documented then, and here could be a good place to do so.

Reply:
Fair enough. This whole thing is not near it's final from though, still figuring out a lot of stuff to make it more testable.